pyproject.toml

```diff
@@ -79,6 +79,7 @@ dependencies = [
     "ruamel.yaml>=0.18",
     "pefile>=2023.2.7",
     "pyelftools>=0.31",
+    "pyghidra>=3.0.0",
```

Comment:
can we remove this from the standard requirements and add a new ghidra install option?